configure

```diff
@@ -679,9 +679,9 @@ case "$CFG_RELEASE_CHANNEL" in
 esac
 
 # Adjust perf and debug options for debug mode
-if [ -n "$CFG_ENABLE_DEBUG" ]; then
+if [ $CFG_ENABLE_DEBUG -eq 1 ]; then
```

Comment:
They don't warn; these had to change because of the addition of `eval $V = 0` on L266